src/UserService.ts

```diff
@@ -371,6 +386,20 @@ class UserService extends Base {
     return (json as unknown) as SubscriptionInfo;
   }
 
+  async getUserSubscriptionOfflineActivationCode(username: string, subscriptionId: string, offlineCodeActivationData: ActivationCodeData): Promise<OfflineActivationCode> {
+    const { apiEndpointAddress, fetch } = this.lensPlatformClient;
+    const url = `${apiEndpointAddress}/users/${username}/subscription-seats/${subscriptionId}/activation-code?accessToken=a${offlineCodeActivationData.accessToken}&refreshToken=${offlineCodeActivationData.refreshToken}&idTokenTest=${offlineCodeActivationData.idToken}`;
```

Comment:
It's unsafe to pass sensitive data in url. (Yeah, need to change the backend implementation really).

Reply:
We can keep activation code on frontend then as it's generation requires token data which we can't get on backend

Reply:
Not sure if passing in request headers will be much different

Reply:
Passing in headers is considered safe (that's how Authorization header is passed normally, for example). But yeah, generating it online (like currently do AFAIK) is also fine.

Reply:
Yes we can keep the code generation on client at least for now.

Reply:
Headers are encrypted if it's via HTTPs (afaik)

Reply:
Moved to a single POST and post body